### ISSUE TITLE
Reputation System for Law Protection

### ISSUE THREAD

**Issue by cbbcbail** [wetube/bitcloud#27]
While reading and reviewing the paper on the Laws and the Proof of Bandwidth, I realized that there was a hole in the plan where there was no explanation for the system that would be used to monitor and facilitate people's reputation. 

I have used StackExchange for a while and have always greatly admired their brilliant and seemingly perfectly implemented Reputation System. I have suggested that we use a similar system for monitoring people's actions and determining whether rewards or punishments should be carried out. 

I am interested to hear what other people think about this idea and would be willing to further expand upon and develop it if people are interested and think that it would work.

In conclusion, the flaw is the lack of a defined system by which the network can tell how a person is doing in terms of following the Laws/Rules. I have a suggestion for a solution in a Reputation System much like the on used at StackExchange.
